sourcemash/frontend/templates/category.tpl

```diff
@@ -19,6 +19,8 @@
           <span class="badge"><a href="#/categories/<%=item.get('category_2')%>"><%=item.get('category_2')%></a></span>
           <h4><a href="<%= '#/items/' + item.get('id') %>"><%= item.get('title') %></a></h4>
         </span>
+        <span class="waves-effect btn-floating red"><i class="mdi-action-thumb-down downvote"></i></span>
```

Comment:
We should be using class="row" and class"col m12" etc. to make it look nicer on the phone. Up to you if you want to do that here or just add it as another issue.